openmetadata-ui/src/main/resources/ui/src/utils/ClassificationUtils.tsx

```diff
@@ -80,7 +80,7 @@ export const getCommonColumns = (): ColumnsType<Tag> => [
           <Badge
             className="badge-grey"
             count={t('label.disabled')}
-            data-testid="disabled"
+            data-testid={`tag-disabled-${record.name}`}
```

Comment:
Don't we need to update test to check for this testID?

Reply:
No, this badge is being used with the Table Tag Items. So no need to change the test here.

Reply:
Ok so you eliminate possiblity of 2 disabled elements

Reply:
Yes!